setup.sh

```diff
@@ -165,6 +166,27 @@ function cmd_output_manager()
   return "$?"
 }
 
+# KW used git to track saved configs from kernel-config-manager.
+# It changed so this function removes the unused .git folder from
+# the dot_configs_dir
```

Comment:
What happens if we do not remove the unused .git?
Also, add a TODO as a reminder to drop this one day.

Reply:
`kw k` will function as intended. The .git folder will be superfluous and it will not be updated but the functionality of `kw k` will stay the same.